src/log.c

```diff
@@ -13,21 +14,18 @@
 
 bool verbose = false;
 
-FILE *log_file;
 extern config_t *ini_config;
+extern char *log_filepath;
 
-void construct_log_filename(char **log_filename, char *log_filepath)
+void construct_log_filename(struct db_t *db_t, const char *log_name)
```

Comment:
Why this change? Does `construct_log_filename` for something other than the filename?

Reply:
> Why this change? Does `construct_log_filename` for something other than the filename?

You are right, this was a redundant change, ill fix it.